pipelines/managed/rh-advisories/README.md

```diff
@@ -28,6 +28,10 @@ the rh-push-to-registry-redhat-io pipeline.
 | trustedArtifactsDebug           | Flag to enable debug logging in trusted artifacts. Set to a non-empty string to enable                                             | Yes      | ""                                                        |
 | dataDir                         | The location where data will be stored                                                                                             | Yes      | /var/workdir/release                                      |
 
+## Changes in 2.0.3
```

Comment:
This should be 2.0.4, because you forgot to upversion the task in the previous change.

Reply:
fixed